omg/entities/projectile.py

```diff
@@ -61,13 +61,14 @@ def create(cls, init_px, init_py, angle) -> Projectile:
         )
 
 
-class IceShardFactory(ProjectileFactory):
+class IceElementFactory(ProjectileFactory):
```

Comment:
in your current design this does not create a projectile anymore (which means it is not a ProjectileFactory) and you only used their class name (overkill to create a class if you only need a string).